docs/website/packages/codegen/index.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 1
+title: Codegen
+---
+
+import DocCardList from '@theme/DocCardList';
+
+[![npm](https://img.shields.io/npm/v/@bonfhir/codegen)](https://www.npmjs.com/package/@bonfhir/codegen)
+
+```bash npm2yarn
+npm install @bonfhir/codegen
+```
+
+The `codegen` package's purpose is the solve the problem of supporting multiple versions of FHIR in the Bonfhir repo. It generate typescript code that will be ready to interact with Fhir, by loading complex the Fhir definitions and integrating them into the code. It simplifies the process of creating the necessary code structure to ensure that the your package or application can correctly handle and use Fhir Resource data. FHIR structure definition files are stored in `packages/codegen/definitions/fhir`.
```

Comment:
There is a typo here.

Also - we could add a link to the GitHub repo code for the definitions (https://github.com/bonfhir/bonfhir/tree/main/packages/codegen/definitions/fhir)